src/Contexts/AuthContext.tsx

```diff
@@ -10,10 +10,12 @@ import {
   User,
   UserInfo,
   updateProfile,
+  GoogleAuthProvider,
```

Comment:
This can be removed since we are not using google sign in

Reply:
Yes!